betty/json/linked_data.py

```diff
@@ -14,6 +16,19 @@
     from betty.model.ancestry import Link
 
 
+def carry_definitions(
```

Comment:
Should we instead introduce a `Schema` class to manage this? Then we could fold the schema name into it for referenceable schemas.